image-cropper/.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[build]
```

Comment:
Please add this project at github action to check it can build or not. even if there are no test, it will check whether there are build error or not.

Reply:
I totally forgot about it. Thank you!